README.md

```diff
@@ -1,5 +1,5 @@
 # Presto
-[![Build Status](https://github.com/prestosql/presto/workflows/CI/badge.svg)](https://github.com/prestosql/presto/actions?query=workflow%3ACI+event%3Apush+branch%3Amaster)
+[![Build Status](https://github.com/prestosql/presto/workflows/checks/badge.svg?branch=master&event=push)](https://github.com/prestosql/presto/actions?query=workflow%3Achecks+event%3Apush+branch%3Amaster)
```

Comment:
My guess is we don't want to have 3 badges for the CI, so I picked one.

Reply:
I think we could actually remove the CI badge. It's less useful for GitHub actions since it switches to a "building" state anytime master is pushed. I'd also rather not publicize flakey tests that happen to fail on master

Reply:
This shouldn't be flaky, since this is `checks` workflow. I am still removing it.